app/presenters/idv/in_person/ready_to_verify_presenter.rb

```diff
@@ -46,14 +46,26 @@ def service_provider_homepage_url
         sp_return_url_resolver.homepage_url if service_provider
       end
 
-      def usps_outage_message_enabled
-        IdentityConfig.store.in_person_usps_outage_message_enabled
+      def outage_message_enabled
```

Comment:
since we're here, can we make this a `?` method?
```suggestion
      def outage_message_enabled?
```